pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/IntermediateResultsBlock.java

```diff
@@ -69,6 +69,7 @@
   private boolean _numGroupsLimitReached;
   private int _numResizes;
   private long _resizeTimeMs;
+  private long _threadTime;
```

Comment:
suggest: `threadCpuTimeMicros` instead of `threadTime` (Hint: Always add the units in variable name)

Reply:
done